### DISCUSSION TITLE
前端文件是打包进程序了？

### DISCUSSION THREAD

**Discussion by aliuq** [AlistGo/alist#4042] [Q&A]
我在docker容器里面没找到前端文件目录，现在发现个问题，同是 3.15.0 版本，前端界面不一致

**Comment by Rektyfikowany** [comment]
前端由[alist-org/alist-web](https://github.com/alist-org/alist-web)来维护，后端编译时会打包[public/dist](https://github.com/alist-org/alist/tree/main/public/dist)中的网页，
可以描述一下界面不一致的现象和复现过程吗

**Reply by aliuq** [reply]
左边的是Windows alist，右边的是docker镜像中的，都是3.15.0版本，但是配置参数却不一样，不知道是不是缓存了旧数据？

![image](https://user-images.githubusercontent.com/11707934/230309898-c7f44a71-30f4-4ae1-9c92-b6db8f1b7a1c.png)

**Reply by Rektyfikowany** [reply]
左侧多出的异常参数应该来自v2版本的data文件，建议检查windows下的配置文件，v2与v3的data互不兼容
